dali/test/python/test_operator_multipaste.py

```diff
@@ -232,7 +237,8 @@ def test_operator_multipaste():
         [4, 2, (128, 128), (128, 128), False, False, False, False, False, types.UINT8],
         [4, 2, (128, 128), (128, 128), False, False, False, False, False, types.INT16],
         [4, 2, (128, 128), (128, 128), False, False, False, False, False, types.INT32],
-        [4, 2, (128, 128), (128, 128), False, False, False, False, False, types.FLOAT]
+        [4, 2, (128, 128), (128, 128), False, False, False, False, False, types.FLOAT],
```

Comment:
Do we have tests with out-of-bounds/clipping?

Reply:
Added those tests now